java/bench/core/src/java/org/apache/orc/bench/core/convert/GenerateVariants.java

```diff
@@ -221,8 +221,8 @@ public static BatchReader createReader(Path root,
                                          long salesRecords) throws IOException {
     switch (dataName) {
       case "taxi":
-        return new RecursiveReader(new Path(root, "sources/" + dataName), "csv",
-            schema, conf, CompressionKind.ZLIB);
+        return new RecursiveReader(new Path(root, "sources/" + dataName), "parquet",
+            schema, conf, CompressionKind.NONE);
```

Comment:
I believe this should be recovered along with https://github.com/apache/orc/pull/1141/files#r881201813

Reply:
What do you mean be recovered? The CSV files are no longer available

Reply:
https://github.com/apache/orc/pull/1141#issuecomment-1136712610

Reply:
I overlooked this sentence, `updated taxi dataset source and schema (the old one no longer exists)`, at the initial review.

Reply:
So, never mind about Parquet downloading part. I'll revise those comments.